Bottom: bb08ae265f24eda36f93836ca23a715bf4c5f66b
Top:    2e49cffa99107cc5dddda5b57cf335911350a8cb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-12 04:47:44 +0100

Refresh of try-clipping-polygons-to-the-s

---

diff --git a/src/draw.c b/src/draw.c
index 29e4cec..ae6d6f0 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -2289,7 +2289,7 @@ DrawPlainPolygon (LayerTypePtr Layer, PolygonTypePtr Polygon, const BoxType * cl
           noholes_piece = Polygon->NoHoles;
           do
             {
-              POLYAREA *tmp_b, *tmp_f, *clipped_result;
+              POLYAREA *tmp_b, *tmp_f;
 
               tmp_b = noholes_piece->b;
               tmp_f = noholes_piece->f;
